app/models/admin_ability.rb

```diff
@@ -118,7 +118,10 @@ def signed_in_with_organizer_role(user, conf_ids_for_organization_admin = [])
     can :manage, Commercial, commercialable_type: 'Conference',
                              commercialable_id: conf_ids
     can :manage, Registration, conference_id: conf_ids
-    can :manage, RegistrationPeriod, conference_id: conf_ids
+    can :manage, RegistrationPeriod do |registration_period|
+      conference = registration_period.conference
+      conf_ids.include?(conference.id) && conference.tickets.for_registration.any?
```

Comment:
This will fail when creating the RegistrationPeriod, but not when accesing the route, that was more my concern.